samples/VanillaJSTestApp/e2eTests/TestCredential.ts

```diff
@@ -0,0 +1,49 @@
+import { ClientSecretCredential, AccessToken } from "@azure/identity";
+import axios from "axios";
+const labApiUri = "https://msidlab.com/api"
+
+export class TestCredential {
```

Comment:
nit: This is more of a LabClient than a TestCredential

Reply:
makes sense, I can rename